librz/core/canalysis.c

```diff
@@ -110,6 +120,21 @@ static char *getFunctionName(RzCore *core, ut64 addr) {
 	return (flag && flag->name) ? strdup(flag->name) : NULL;
 }
 
+static char *getFunctionNamePrefix(RzCore *core, ut64 off, const char *name, bool prefix) {
```

Comment:
parts of these two functions was just removed/is going to be removed by @08A PR.

Reply:
Addressed this thing, also removed `prefix` parameter since it's not used in the new code after @08A changes.